requirements/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "torchgeo",
```

Comment:
I'm curious how many of these values are required. Here we're only using this file to track dependencies. We aren't going to publish torchgeo on npm.